validators.go

```diff
@@ -50,7 +51,39 @@ var EncodingValidator = &Validator{
 				}
 			}
 
-			return ErrBadEncoding
+			return &ValidationError{
+				Err: ErrBadEncoding,
+				Context: Context{
+					"badRunes": bad,
```

Comment:
It appears most the context fields used elsewhere use underscores, not camel case. Since these are map keys I don't personally think they need to adhere to the camel case idiom, however they should be either or.